packages/js/src/helpers/classicEditor.js

```diff
@@ -67,19 +68,21 @@ class RegisteredComponentsContainer extends wpComponent {
 export function renderClassicEditorMetabox( store ) {
 	const localizedData = getL10nObject();
 	containerRef = createRef();
-
+	const metaboxLocationContext = { locationContext: "classic-metabox" };
```

Comment:
Maybe not bind the variable name to the location. Then it is clear you can expose more data like this in the same root context (object).
```suggestion
	const classicMetaboxContext = { locationContext: "classic-metabox" };
```